i18n-module.php

```diff
@@ -111,7 +111,7 @@ public function __construct( $args ) {
 			return;
 		}
 
-		$this->locale = get_locale();
+		$this->locale = get_admin_locale();
```

Comment:
Because `get_admin_locale` is in a class you have to call it this way `$this->get_admin_locale()`